common/common.go

```diff
@@ -22,6 +22,17 @@ func init() {
 
 func initEqNameToWeapon() {
 	eqNameToWeapon = make(map[string]EquipmentElement)
+	// TODO: these seem to be unmapped
```

Comment:
Yeah they are currently umapped since that should turn them into `EqUnknown` as it's the default value.

These weapons can't be obtained in-game with the exception of sensorgrenade, which could be obtained in some of the co-op missions, but we don't care about that.

It would be better to map them to it explicitly and then check if eqNameToWeapon contains a mapping and if not emit a warning. But we don't have easy access to dispatching events from this package right now.

See TODO in `MapEquipment`

Reply:
I'll remove these again and let this be handled in `MapEquipment` :slightly_smiling_face: